app/views/course/discussion/topics/_topics.html.slim

```diff
@@ -0,0 +1,4 @@
+- @topics.each do |topic|
+  = render partial: topic.actable_type.demodulize.underscore, locals: { topic: topic }
```

Comment:
You mean `= render topic.actable, prefix: 'discussion_topic'`  ?

It looks cleaner, but one of my concern is that then the specific partials are in different folders, very hard for them to share other common partials ( In my concrete implementation there is a _answer_info partial shared by the specific partials ).